.github/workflows/build_and_publish_doc.yml

```diff
@@ -23,20 +23,22 @@ jobs:
   publish:
     needs: [call-build-html-doc, call-build-schema-page]
     runs-on: ubuntu-20.04
+    permissions:
+      attestations: write
```

Comment:
Do we need `write` for all steps below? Is it possible to grant permissions for each step?

Reply:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

I did not find in documentation anything about permission to specific step, only for a job.